engine/pom.xml

```diff
@@ -51,7 +51,7 @@
     <odfdom-java.version>0.8.6</odfdom-java.version>
     <commons-collections4.version>4.4</commons-collections4.version>
     <commons-math3.version>3.6.1</commons-math3.version>
-    <xmlbeans.version>3.1.0</xmlbeans.version>
+    <xmlbeans.version>5.2.0</xmlbeans.version>
```

Comment:
Is this needed for the POI library update?

Reply:
yes @peterrinehart , xmlbeans: The Apache XMLBeans library, used by POI to process XML-based data in OOXML documents.